grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.tsx

```diff
@@ -18,23 +18,29 @@ import styles from 'containers/OutgoingWebhook2Form/OutgoingWebhook2Form.module.
 const cx = cn.bind(styles);
 
 interface OutgoingWebhook2FormProps {
-  id: OutgoingWebhook2['id'] | 'new';
+  id: OutgoingWebhook2['id'];
+  action: 'new' | 'update';
   onHide: () => void;
   onUpdate: () => void;
 }
 
 const OutgoingWebhook2Form = observer((props: OutgoingWebhook2FormProps) => {
-  const { id, onUpdate, onHide } = props;
+  const { id, action, onUpdate, onHide } = props;
 
   const store = useStore();
 
   const { outgoingWebhook2Store } = store;
 
-  const data = id === 'new' ? { is_webhook_enabled: true, is_legacy: false } : outgoingWebhook2Store.items[id];
+  const data =
+    id === 'new'
```

Comment:
is `id === 'new'` still possible given the change above in the `OutgoingWebhook2FormProps` interface?

Reply:
Double-checking, it looks like we still need that (to set the default values on create), restoring it :+1: